src/engine/source/metrics/input_files/test-metrics-config.json

```diff
@@ -0,0 +1,54 @@
+[   
```

Comment:
is this file necessary? what would it be used for?

Reply:
You have to separate the test metrics from those that are for the modules. Eventually this will disappear, but for now when you run the list command you only have to see the metrics of the modules.